crypto/x509/v3_alt.c

```diff
@@ -71,8 +71,13 @@ STACK_OF(CONF_VALUE) *i2v_GENERAL_NAME(X509V3_EXT_METHOD *method,
 
     switch (gen->type) {
     case GEN_OTHERNAME:
-        if (!X509V3_add_value("othername", "<unsupported>", &ret))
-            return NULL;
+        if (OBJ_cmp(gen->d.otherName->type_id, OBJ_nid2obj(NID_id_on_SmtpUTF8Mailbox)) != 0) {
```

Comment:
I think here (and below) we should handle the `==` case (`SmtpUtf8Mailbox`) first, and always leave the default (unsupported) case to last.

Reply:
Done.